hutch_python/ipython_session_timer.py

```diff
@@ -0,0 +1,94 @@
+"""
+This module modifies an ``ipython`` shell to automatically close if it has been
+idle for 48 hours. Prior to starting hutch-python the user is alerted that the
```

Comment:
Technically now that you've let us configure this, this module docstring is out of date